testing/xfce4-sensors-plugin/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: Carlo Landmeter <clandmeter@gmail.com>
+# Maintainer:
```

Comment:
Yes, some devs unfortunately do this. I personally disagree with this practice, but it seems that there's no consensus. We have to discuss this in dev team.

Shiz, since some (even core) devs sometimes add new pkgs to testing w/o claiming maintainership, I think that it'd be fair to accept this PR (if it's otherwise okay, I haven't looked at it). WDYT?